thisissandip/gutenberg
# Add a dot before the parent block selector when there are unsaved changes in the Reusable block

## Description

While working on https://github.com/WordPress/gutenberg/pull/32464# it was [proposed](https://github.com/WordPress/gutenberg/pull/32464#issuecomment-886911336) to add a dot before the parent block selector to guide the user about the unsaved changes in the Reusable block.

## How has this been tested?
1. Select a Reusable Block
2. Make changes 
3. A dot should be visible before parent block selector Icon

## Screenshots 

https://user-images.githubusercontent.com/69596988/127275144-f16715fc-b687-461a-95c1-871e2c0a6206.mov

## Types of changes

New feature 


## Checklist:
